examples/step-28/step-28.cc

```diff
@@ -158,9 +158,9 @@ namespace Step28
   //
   // At present, material data is stored for 8 different types of
   // material. This, as well, may easily be extended in the future.
-  MaterialData::MaterialData (const unsigned int n_groups)
+  MaterialData::MaterialData (const unsigned int n_groups_)
     :
-    n_groups (n_groups),
+    n_groups (n_groups_),
```

Comment:
`n_groups` was unused in Release mode since we only use it in `Assert`s. So this is indeed just for shutting up the compiler.